projects/arlas-components/src/lib/components/results/result-detailed-grid/result-detailed-grid.component.ts

```diff
@@ -146,8 +157,7 @@ export class ResultDetailedGridComponent implements OnChanges, OnDestroy {
 
     if (this.useHttp) {
       this.isLoading = true;
-      this.http.get(this.gridTile.urlImages[this.currentImageIndex], { responseType: 'blob' })
-        .pipe(take(1))
+      this.http.get(this.gridTile.urlImages[this.currentImageIndex], {headers: {[PROTECTED_IMAGE_HEADER]: 'true'}, responseType: 'blob' })
```

Comment:
take(1) is no longer needed ?

Reply:
That is something I was unsure about. I added it because I had in mind we had to unsubscribe to observables as much as possible, but this is an http request, so shouldn't it unsubscribe itself as soon as the answer is received ?

Reply:
In general, you should unsubscribe from an observable when a component is destroyed.

You don't have to unsubscribe from [HttpClient](https://angular.io/api/common/http/HttpClient) observables because they unsubscribe automatically after the server request responds or times out. Most developers choose not to unsubscribe. None of this guide's examples unsubscribe.

Reply:
LGTM then